### PR TITLE
CryoEngines and related extras

### PR DESCRIPTION
**Pull request by ChrisAdderley**
This should be getting to the end of these. Moves to self hosting and adds a previously un-indexed mod.

### CODE REVIEW

File: NetKAN/CryoEngines-LFO.netkan [hunk 1]
```diff
@@ -1,16 +1,5 @@
 {
     "spec_version": "v1.4",
     "identifier": "CryoEngines-LFO",
-    "$kref": "#/ckan/spacedock/709",
-    "$vref": "#/ckan/ksp-avc/GameData/CryoEngines/Versioning/CryoEngines.version",
-    "name": "Cryogenic Engines Extras: LFO Patch",
-    "abstract": "Optional patch to make the engines burn KSP's normal liquid fuel and oxidizer mixture instead of the custom liquid hydrogen and oxidizer mixture that they use by default.",
-    "license": "CC-BY-NC-SA-4.0",
-    "x_netkan_epoch": 1,
```
Comment (HebaruSan) [inline NetKAN/CryoEngines-LFO.netkan]: This `x_netkan_epoch` property was lost in the transition; it needs to be preserved, or the bot will treat newer versions as old versions.

Comment (ChrisAdderley) [reply NetKAN/CryoEngines-LFO.netkan]: So it's an override on version numbers? eg 
`1.0.` with `x_netkan_epoch": 1` would count as newer than say `14.0` with `x_netkan_epoch": 0`

Comment (HebaruSan) [reply NetKAN/CryoEngines-LFO.netkan]: Yes. In this case it was added in #1911 because the 1.81 release was followed by 1.9, and 9 < 81.

Comment (ChrisAdderley) [reply NetKAN/CryoEngines-LFO.netkan]: Mm ok. This is exactly why I couldn't support CKAN before validating and managing all this meta myself. Presumably there would have been a typo in a package somewhere having version as `1.81` versus `1.8.1` but to my knowledge this mod has never exceeded version 0.6.6 so... 🤷‍♂️ 

I restored the field in the host repo.

Comment (HebaruSan) [reply NetKAN/CryoEngines-LFO.netkan]: Sorry, typo on my part; the relevant versions were 0.1.81 and 0.1.9. I got confused by where one number stopped and the next began.

image

**Comment by HebaruSan** [comment]
#rebuild

File: NetKAN/CryoEngines-LFO.netkan (hunk 1 above)
Comment (ChrisAdderley) [reply NetKAN/CryoEngines-LFO.netkan]: Ah yeah, same issue I guess - likely a borked upload to I guess, KerbalStuff back then.

**Comment by HebaruSan** [comment]
Looks good now. Thanks for another update!

**Pull request merged**